chat-angular/src/environments/environment.ts

```diff
@@ -0,0 +1,9 @@
+export const environment = {
+  production: false,
+  APP_ID: 'REPLACE_APP_ID',
+  AUTH_KEY: 'REPLACE_APP_AUTH_KEY',
+  AUTH_SECRET: 'REPLACE_APP_AUTH_SECRET',
+  CONFERENCE_SERVER_ENDPOINT: 'wss://REPLACE_APP_JANUS_DOMAIN:8989',
```

Comment:
for a sake of simplicity, let's  remove CONFERENCE_SERVER_ENDPOINT, CHAT_ENDPOINT, API_ENDPOINT from configs and use default one. Shared users do not need this to change